manifests/lenster.json

```diff
@@ -1,21 +1,21 @@
 {
-  "appId": "lenster",
-  "name": "Lenster",
-  "description": "Lenster is a composable, decentralized, and permission-less social media web app built with Lens Protocol 🌿",
+  "appId": "Hey",
```

Comment:
There is no hard rule set on this yet... but usually I've seen App IDs all lowercased. @bigint can you please confirm what you used in your queries?

Reply:
You can see here that the app id is in Upper case https://arweave.app/tx/kk_or6Patce9-cg80YaK8OiT6Nvn3Gsg6Sm_WuZ8wlc 
![image](https://github.com/lens-protocol/lens-share/assets/91475837/61e6b2ab-b0e2-4ef5-83f6-900ccee02b0b)

Reply:
oh good point!

I think I should add a little slack to the filtering by `appId` logic... I can see people crafting links with lowercased `hey`